Makefile

```diff
@@ -153,6 +156,7 @@ deps: runtime
 		mkdir -p ~/.luarocks; \
 		$(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.OPENSSL_LIBDIR $(addprefix $(ENV_OPENSSL_PREFIX), /lib); \
 		$(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.OPENSSL_INCDIR $(addprefix $(ENV_OPENSSL_PREFIX), /include); \
+		$(ENV_LUAROCKS) config $(ENV_LUAROCKS_FLAG_LOCAL) variables.PCRE_INCDIR $(addprefix $(ENV_PCRE_PREFIX), /include); \
```

Comment:
The variable `ENV_PCRE_PREFIX` will only be set in darwin, so this command should be conditional.

Reply:
Thank you for the reminder.Actually I found `pcre.h` in `/usr/include` in my Ubuntu vm.I will wrap this line by an `ifeq` condition for darwin.What do you think of the condition i am going to add?Is it enough?

Reply:
Just check if this variable exists.